data/wanderer/wanderers start.txt

```diff
@@ -528,12 +528,12 @@ mission "Wanderers: Defend Vara Ke'sok"
 			branch hint
 				has "Wanderers: Defend Vara Ke'sok Hint: done"
 			`Soon after you land in the spaceport, there is a flurry of activity as many of the Wanderers fly to their ships, which begin to take off from the planet. In the sky, you see bright flashes that may be distant weapons fire or explosions.`
-			`	You are buffeted by wind as one of the Wanderers lands next to your ship and says, "Unfettered [raider, pirate] fleet comes. We have put food on freighter, hope they take [offering, gift] and leave. If not [appeased, satisfied] you help convince them to leave. Yes?"`
+			`	You are buffeted by wind as one of the Wanderers lands next to your ship and says, "An Unfettered [raider, pirate] fleet comes. We have put food on a freighter, hoping they take the [offering, gift] and leave. If they're not [appeased, satisfied], will you [compel, influence] them to leave?"`
```

Comment:
I thought about this, but I ignored it because I thought the Wanderer, while in flight, was trying to say as little as possible to convey you the information. It was just a little bit of variation.

Reply:
My interpretation is that these Wanderers were less skilled at speaking Hai than the ambassadors and scientists, and the translator correctly translated their grammatically-incorrect sentences.

Reply:
I have reverted the text and added this: `Their words are so quick, your translator can barely keep up.`